pkg/ds/daemon_set.go

```diff
@@ -310,6 +323,17 @@ func (ds *daemonSet) WatchDesires(
 	return errCh
 }
 
+func (ds *daemonSet) reportEligible() error {
+	eligible, err := ds.EligibleNodes()
+	if err != nil {
+		return util.Errorf("Error retrieving eligible nodes for daemon set: %v", err)
+	}
+	name := "total_" + ds.ID().String()
```

Comment:
hyphen somehow seems like a better choice to me but not sure i really know why i think that